From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Thu, 13 Aug 2015 12:57:01 +0200
Subject: Doxygen fix for cmap_iter_next()

Remove the extra cmap_ prefix of the iter_handle parameter.
---
 include/corosync/cmap.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/corosync/cmap.h b/include/corosync/cmap.h
index b43dcea..b10993b 100644
--- a/include/corosync/cmap.h
+++ b/include/corosync/cmap.h
@@ -283,7 +283,7 @@ extern cs_error_t cmap_iter_init(cmap_handle_t handle, const char *prefix, cmap_
  * length of returned value and/or type is returned.
  *
  * @param handle cmap handle
- * @param cmap_iter_handle handle of iteration returned by cmap_iter_init
+ * @param iter_handle handle of iteration returned by cmap_iter_init
  * @param key_name place to store name of key. Maximum length is CMAP_KEYNAME_MAXLEN
  * @param value_len length of value
  * @param type type of value
